@xen-orchestra/lite/src/components/menu/MenuItem.vue

```diff
@@ -13,7 +13,14 @@
     </MenuTrigger>
     <AppMenu v-else shadow :disabled="isDisabled">
       <template #trigger="{ open, isOpen }">
-        <MenuTrigger :active="isOpen" :color="color" :icon="icon" @click="open">
+        <MenuTrigger
+          :active="isOpen"
+          :busy="isBusy"
+          :disabled="isDisabled"
+          :color="color"
+          :icon="icon"
+          @click="(ev) => (!isDisabled ? open(ev) : undefined)"
+        >
```

Comment:
This trigger is meant to open the submenu.

We shouldn't disable it when busy (BTW, in which case an item which opens a submenu could be busy?).

As discussed before, this item shouldn't be disabled when the menu itself is disabled, since we want to be able to view the submenu items, even if disabled.